proto/src/determined/model/v1/model.proto

```diff
@@ -49,7 +50,7 @@ message Model {
   // Username of the user who created this model.
   string username = 10;
   // The id of the workspace associated with this model.
-  optional int32 workspace_id = 14;
+  int32 workspace_id = 14;
```

Comment:
the changes related to this lgtm! Thanks Nick.

Reply:
As part of this PR nick, can you please change type of workspace_id in [PatchModel](https://github.com/determined-ai/determined/blob/44c39c6c5a8b49723f08da0843ce9e7d6e744a91/proto/src/determined/model/v1/model.proto#L80) from optional string workspace_id to optional int32 workspace_id? Thank you for catching this @keita-determined